dotnet-471-dev-pack/test/test.pester.ps1

```diff
@@ -0,0 +1,33 @@
+$__dir=(Get-Item $PSScriptRoot)
+$PLAN_ROOT=(Get-Item $__dir ).parent.FullName
+$CORE_ROOT=(Get-Item $PLAN_ROOT ).parent.FullName
+
+. $CORE_ROOT/results/last_build.ps1
+
+Describe "The NETFXSDK installation when" {
+    Context "is the SDK version 4.7" {
+        $sdk_version="4.7"
+        It "should have the 'Program Files/Windows Kits/NETFXSDK/$sdk_version/Include/um' directory" {
```

Comment:
My personal preference (feel free to ignore :) ) is to state what the behaviour of the system _is_ rather than what it _should_ be

e.g.

` It "has the 'Program Files/Windows Kits/NETFXSDK/$sdk_version/Include/um' directory"`

Reply:
Amended.